headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/graphql/mutation/v1_0/Mutation.java

```diff
@@ -56,27 +56,15 @@ public static void setWarehouseItemResourceComponentServiceObjects(
 			warehouseItemResourceComponentServiceObjects;
 	}
 
+	@GraphQLField
 	@GraphQLInvokeDetached
-	public Response deleteWarehousId(@GraphQLName("id") Long id)
-		throws Exception {
-
-		return _applyComponentServiceObjects(
-			_warehouseResourceComponentServiceObjects,
-			this::_populateResourceContext,
-			warehouseResource -> warehouseResource.deleteWarehousId(id));
-	}
-
-	@GraphQLInvokeDetached
-	public Response patchWarehousId(
-			@GraphQLName("id") Long id,
-			@GraphQLName("warehouse") Warehouse warehouse)
+	public Warehouse postWarehous(@GraphQLName("warehouse") Warehouse warehouse)
```

Comment:
@marco-leo a bunch of these methods are misspelled (`*Warehous*` rather than `*Warehouse*`). Is this... intended? There's a lot of them.